tools/clang/test/HLSL/ShaderOpArith.xml

```diff
@@ -2265,16 +1936,16 @@
         RWByteAddressBuffer g_rawBuf : register(u2);
         RWByteAddressBuffer g_rawXchgBuf : register(u3);
 
-        RWBuffer<uint> g_uintBuf : register(u4);
-        RWBuffer<int> g_sintBuf : register(u5);
-        RWBuffer<int> g_xchgBuf : register(u6);
+        RWBuffer<uint2> g_shareBuf : register(u4);
+        RWBuffer<uint2> g_shareXchgBuf : register(u5);
 
-        RWTexture1D<uint> g_utexBuf : register(u7);
-        RWTexture1D<int> g_stexBuf : register(u8);
-        RWTexture1D<int> g_xtexBuf : register(u9);
+        RWBuffer<uint> g_uintBuf : register(u6);
+        RWBuffer<int> g_sintBuf : register(u7);
+        RWBuffer<int> g_xchgBuf : register(u8);
```

Comment:
This is a misleading diff. No changes were made to these variables except that they were shuffled down to make room for the groupshared outputs